src/main/java/org/opentripplanner/apis/common/json/FeedScopedIdDeserializer.java

```diff
@@ -1,4 +1,4 @@
-package org.opentripplanner.api.json;
+package org.opentripplanner.apis.common.json;
```

Comment:
Move to `org.opentripplanner.ext.restapi.serialization`

Reply:
I think this is used transitively by Jersey and therefore in theory by all APIs. At least the GTFS API converts the FeedscopedIds to strings before using the serialization layer.